Added various soups (#367)
LukeSmithxyz/based.cooking: src/finnish-fish-soup.md

# Finnish fish soup

Traditional Finnish fish soup usually served with rye bread.
Originally from a finnish book _Saiturin Keittiö_ (The Cheapskates Kitchen) from 1992. 

> Cost of ingredients originally 30 Finnish mark or about 8€ adjusted for inflation.

Reviewer: ```suggestion
    ⏲️ Time: 50 min
    🍽️ Servings: 4
    💲 Cost ~8€
```
Need this for all the recipes, you can be creative with the emojis!

- 🍽️ Servings: 4
- 💲  Cost ~8€

## Ingredients

- 600g of small fish (eg. perch) or fish offal (heads, fins, ...)
- 6 midsized potatoes
- small packet of frozen spinach or 50g of parboiled fresh spinach
- 1 tbsp of wheat flour

> - 2 egg yellows

Reviewer 2: ```suggestion
- 2 egg yolks
```

> - 1/2 dl of cream

Reviewer 2: Can't say I've seen a `dl` in a cooking recipe before. Is this a `decilitre` or something else? Might be best to use more standard measurement units.

Author: dl does indeed stand for decilitre. I think it's the norm in europe. Happy to add another measurement unit if you deem it necessary.

Reviewer 2: Maybe leave it for now, and we can see what Luke's opinion is. All I know is that in Canada it is not common outside of science class :)

Reviewer 3: `dl` is very much used indeed, but if it is rather unknown over the pond then we should add an alternative in parentheses, though I do not myself know what that unit would be.

Reviewer 4: In the states you would say "1/4 cup" (~20% more volume) or "3 tbsp" (~10% less volume). "50mL" is probably a better internationally recognizable measure, most Americans will not have heard the word "deciliter" since high school but our measuring cups have secondary milliliter legends.

Reviewer 5: @hallzy dL (decilitres) are very commonly used in Europe and probably elsewhere. [Deci](https://en.wikipedia.org/wiki/Metric_prefix#List_of_SI_prefixes) comes from Latin *decimus*, which means tenth. It's also where the word *decimation* comes from; cutting down something by one-tenth. Americans are probably not going to be aware of this, so I would also recommend "50 mL" if it poses a problem to reading comprehension by American readers.

Reviewer 2: @Holsterbau I am aware of what a dL is, i've just never seen it on a recipe before.

Reviewer 5: @hallzy Right. Regardless, I agree with your point. Litres and millilitres are going to be much more obvious to our non-metric friends.

- a small lump of butter

## Directions

> 1. Boil the fishes for 30 min in 12 dl of water

Reviewer 2: `dl` again. Refer to my above message

Reviewer 5: This should be `1.2 L`. Strange way of using decilitres.

2. Sieve the soup and gut the fishes
3. Put the fishes back into the soup
4. Peel and cut the potatoes
5. Put them into the soup and boil until well done
6. Add the spinach and let it boil 
7. Mix the flour in a dash of cold water and mix with the soup and let it boil
8. Remove the pot from the stove

> 9. Add cream, egg yellows, whitepepper and a small lump of butter

Reviewer 2: ```suggestion
9. Add cream, egg yolks, white pepper and a small lump of butter
```

> 
> 
> 

Reviewer: ```suggestion
```
too much whitespace
